.build/webpack.common.js

```diff
@@ -54,18 +52,6 @@ module.exports = (env, argv) => {
             }
           ]
         },
-        {
```

Comment:
took it out of common and stuck it in the prod config. this way local dev still shows up fine

Reply:
It would be good to try to replicate the prod setup in dev it at all possible i.e. load the pf stylesheet from a cdn.

Reply:
I added a `start:prod` target which will serve up the prod build content so you can load it in `application-services-ui`

Reply:
We want to keep the `start:dev` target working for dev - the right way is to just not include the styles in the federated module.